src/commissaire_http/handlers/clusters.py

```diff
@@ -173,6 +179,33 @@ def create_cluster(message, bus):
         return return_error(message, error, JSONRPC_ERRORS['INVALID_REQUEST'])
 
 
+def delete_cluster(message, bus):
+    """
+    Deletes an existing cluster.
+
+    :param message: jsonrpc message structure.
+    :type message: dict
+    :param bus: Bus instance.
+    :type bus: commissaire_http.bus.Bus
+    :returns: A jsonrpc structure.
+    :rtype: dict
+    """
+    try:
+        name = message['params']['name']
+        LOGGER.debug('Attempting to delete cluster "{}"'.format(name))
+        bus.request('storage.delete', params=[
+            'Cluster', {'name': name}])
+        return create_response(message['id'], [])
+    except _bus.RemoteProcedureCallError as error:
+        LOGGER.debug('Error deleting cluster: {}: {}'.format(
+            type(error), error))
```

Comment:
This may be broader than just this PR, but `type(error)` here is always going to be `RemoteProcedureCallError`.  More useful for debugging would be: `error.data.get('exception', error.__class__.__name__)`